mtp_api/apps/credit/serializers.py

```diff
@@ -145,3 +149,18 @@ def create(self, validated_data):
         user = self.context['request'].user
         validated_data['user'] = user
         return super().create(validated_data)
+
+
+class CreditsGroupedByCreditedSerializer(serializers.Serializer):
+    logged_at = serializers.DateField()
+    owner = serializers.IntegerField()
+    username = serializers.SerializerMethodField()
```

Comment:
might be worth calling this something more like 'owner name' or whatever cos it's not the owner's username